corehq/apps/registration/templates/registration/create_new_user.html

```diff
@@ -6,8 +6,8 @@
 {% block js-inline  %}{{ block.super }}
     {% url "eula_basic" as eula_url %}
     {% include 'hqstyle/partials/load_into_modal.html' with modal_id="eulaModal" load_from=eula_url %}
+    {% if not hide_password_feedback %}
```

Comment:
renamed to `hide_password_feedback` (and use `not`) so password feedback is enabled if the `hide_password_feedback` template variable is undefined